sql/moz-fx-data-shared-prod/telemetry_derived/experiment_search_aggregates_base/view.sql

```diff
@@ -53,7 +52,6 @@ FROM
     )
   ) AS unnested_search_counts
 GROUP BY
-  timestamp,
```

Comment:
Grouping on timestamp was introduced as part of the changes in https://github.com/mozilla/bigquery-etl/pull/1721/, however this caused a significant increase in new rows which is resulting in memory exceeded errors. Grouping on timestamp basically retains every row a search has been done, while we are actually interested in the window aggregates. `timestamp` is not strictly necessary, it was previously used by the hourly table for partitioning but instead `window_start` can be used for this. 

I believe this change doesn't necessarily require changes to the underlying table (we could just ignore `timestamp`), but since the tables are small, and freshly created this week I think with changes still being expected, I'd go ahead and fix them once approved.